src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.html

```diff
@@ -1,13 +1,10 @@
-<div class="form-check form-switch">
-    <input
-        class="form-check-input"
-        type="checkbox"
-        id="enabledDisabledSwitch"
-        [disabled]="!isAdmin && settingsType !== EXERCISE"
-        [(ngModel)]="enabled"
-        (change)="onEnabledChange()"
-    />
-    <label class="form-check-label" for="enabledDisabledSwitch" jhiTranslate="artemisApp.iris.settings.subSettings.enabled-disabled"> Enabled/Disabled </label>
+<div class="btn-group">
+    <div class="btn" [ngClass]="{ 'btn-success selected': enabled, 'btn-default': !enabled }" [class.disabled]="!isAdmin && settingsType !== EXERCISE" (click)="onEnable()">
```

Comment:
Bit nicer way to write the conditional class: [ngClass]="enabled ? 'btn-success selected' : 'btn-default'"